src/components/Hero/index.jsx

```diff
@@ -43,14 +44,14 @@ class Hero extends React.Component {
   }
 
   state = {
-    detailsOpen: false,
```

Comment:
this auto-formatting? I think we want trailing commas

Reply:
:+1:  fixed. Sorry about that